package.json

```diff
@@ -4,6 +4,8 @@
   "scripts": {
     "server": "ts-node server/server.js",
     "server:debug": "node --inspect -r ts-node/register ./server/server.js",
+    "widget": "cd ./ui && npm run dev",
```

Comment:
I tested this, and even if you call `npm run widget` multiple times in a row, calling `cd ./ui` doesn't cause issues. It is idempotent, and thus doesn't seem to be fragile.